packages/components/package.json

```diff
@@ -25,11 +25,13 @@
     "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/*.spec.ts,src/**/*.spec.tsx --ignore src/**/*.stories.ts,src/**/*.stories.tsx",
     "build:types": "tsc --project . --declaration --emitDeclarationOnly",
     "build:watch": "npm run build:js -- --watch",
-    "build:storybook": "build-storybook",
+    "build:storybook": "build-storybook -o storybook-static",
     "deploy:docs": "storybook-to-ghpages --ci",
     "prepack": "npm run build && cp ./package.json dist/",
     "snapshot": "percy-storybook --widths=320,1280",
     "storybook": "start-storybook -p 6006",
+    "storybook:axe": "npm run build:storybook && npm run storybook:axeOnly",
```

Comment:
Is jest-axe still being used? If not, should we remove it?

Reply:
nope, will remove 👍